tests/travis-Debug.patch

```diff
@@ -1,13 +1,13 @@
 --- a/snap/snapcraft.yaml
 +++ b/snap/snapcraft.yaml
-@@ -143,9 +143,8 @@ parts:
-     - golang
```

Comment:
Is this somehow related to ccache?

Reply:
Note this is a diff of a diff, so this isn't an actual change of the resulting file, but rather just some diff fuzz.

Reply:
Right, just the context lines changing... got to get used to diff-squared. Please don't ever go beyond that :)